src/labelPullRequestWithMetricMovement.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
```

Comment:
mostly lifted and shifted from https://github.com/holidayextras/tripapplite/blob/staging/script/util/labelPullRequestWithMetricMovement.js though I renamed a few vars to make use of es6 object shorthand.